rudder-agent/SOURCES/check-rudder-agent

```diff
@@ -267,6 +277,7 @@ then
   CFE_BIN_DIR="${CFE_DIR}/bin"
   LAST_UPDATE_FILE="${CFE_DIR}/last_successful_inputs_update"
   NEED_INIT_FILES="true"
+  check_and_fix_cfengine_keys
```

Comment:
You only added this in the "Once for rudder's cfengine" section, shouldn't we also do it in the "Once for enterprise cfengine" section below? I know we never have before, but if it's useful for one CFEngine it must surely be for the other too.